src/commands/configuration/language.js

```diff
@@ -10,6 +10,7 @@ module.exports = class ConfigLanguage extends Command {
     super({
       name: 'language',
       aliases: ['lang'],
+      category: 'configuration',
```

Comment:
this command has a `parent`, so you don't need to specify the category as it's inherited from the *parent*